### PR TITLE
LPS-120646 adding value from default locale if value from editing loc…

### PR DESCRIPTION
**Pull request by linolaoj**

### CODE REVIEW

**Comment by linolaoj** [comment]
ci:test:sf

**Comment by linolaoj** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/util/fieldSupport.es.js [hunk 1]
```diff
@@ -148,6 +148,18 @@ export const getFieldProperties = (
 					properties[fieldName] = value[editingLanguageId];
 				}
 			}
+			else if (type == 'validation') {
+				if (!value.errorMessage[editingLanguageId]) {
+					value.errorMessage[editingLanguageId] =
+						value.errorMessage[defaultLanguageId];
+				}
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/util/fieldSupport.es.js:155]: @linolaoj, please add a line break after this line and feel free to forward this pull running the command `ci:forward`.

Thanks.

**Comment by linolaoj** [comment]
ci:forward

**Comment by linolaoj** [comment]
ci:forward

**Comment by linolaoj** [comment]
ci:forward

**Comment by natocesarrego** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**